addons/stock_picking_batch/models/stock_picking_batch.py

```diff
@@ -178,6 +186,10 @@ def action_print(self):
         self.ensure_one()
         return self.env.ref('stock_picking_batch.action_report_picking_batch').report_action(self)
 
+    def action_set_done_quantities(self):
+        self.ensure_one()
+        self.picking_ids.action_set_done_quantities()
```

Comment:
Only call `action_set_done_quantities` on pickings that have `show_set_all_done = True`